### PR TITLE
[Docs] clean up list elements and other formatting

### PR DESCRIPTION
**Pull request by samiahmedsiddiqui**
- List elements added incorrectly using HTML tags and due to that it is not showing as per new theme.
- Add rightNav on _indexpage and removed from content

### CODE REVIEW

File: docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ kubectl get services --namespace yb-demo
 Expect output similar to the following:
 
 ```output
-NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                                        AGE
+NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP        PORT(S)                                        AGE
```
Comment (polarweasel) [inline docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md:262]: Where did this one come from? And does it need to be replicated in all the other helm-chart files in this and the other versions?

**Review by polarweasel — APPROVED** [review]
LGTM! One question outstanding, then we should be good to merge.

File: docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md (hunk 1 above)
Comment (samiahmedsiddiqui) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md:262]: @polarweasel On couple of instances, it is already aligned. Let me check all the helm-chart files.

Comment (samiahmedsiddiqui) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md:262]: @polarweasel Applied this change in couple of other instances (especially in helm-chart files) in all versions.

**Comment by polarweasel** [comment]
Thanks for the excellent cleanup with this PR @samiahmedsiddiqui!

**Pull request merged**